chart/kubeseal-webgui/README.md

```diff
@@ -42,7 +42,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `affinity`                                | Add affinity rules to the deployment.             | `{}`                          |
 | `nodeSelector`                            | Add a nodeSelector to the deployment.             | `{}`                          |
 | `displayName`                             | Optional display name for the kubeseal instance   | `""`                          |
-| `resources.limits.cpu`                    | Limits CPU                                        | `100m`                        |
+| `resources.limits.cpu`                    | Limits CPU                                        | `""`                          |
```

Comment:
As we are not setting the value at all, isn't it more a `null` value?

Reply:
Yeah i also thought about that but i think empty is also valid for this table 😄